### PR TITLE
add tasks, enforce jdk version in mvn

### PR DESCRIPTION
**Pull request by okjintao**
Quick changes I made to make maven setup painless as possible. Tasks added.

They're pretty vague right now kind of because I wrote them quickly, kind of because I think we should discuss them too. Maybe if we have time we can triage these tasks and split them up.

### CODE REVIEW

**Comment by okjintao** [comment]
@dafty fwiw I can't add reviewers / labels / etc.

**Comment by okjintao** [comment]
added bip39 passphrase generate / account passphrase verification to be more in line with ark desktop wallet

looked into using futures for calls, new ways of resolving response objects.

File: src/main/java/io/ark/core/responses/Fees.java [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+import static io.ark.core.util.Constants.ARKTOSHI;
+
+import org.json.JSONObject;
+
+import lombok.Data;
+
+/**
+ * This is a proposal for how to deal with all responses from the Ark Node.
+ * This is a frustrating case due to the unit conversions and flat JSON response. 
+ * Being the worse case scenario this is not too bad, and will allow us to use the futreGet added in HttpUtils.
```
Comment (dafty) [inline src/main/java/io/ark/core/responses/Fees.java:12]: We could implement Jackson to take care of HTTP responses.

In a different project, I have all ark-node JSON extend from a base `NodeResponse` object. This makes casting much easier when you know what sort of response you will get back.

`NodeResponse votes = nodeManager.getVotes("AZse3vk8s3QEX1bqijFb21aSBeoF6vqLYE");`

**Review by dafty — APPROVED** [review]

File: src/main/java/io/ark/core/crypto/Crypto.java [hunk 2]
```diff
@@ -23,6 +28,18 @@
...
   private static ECNamedCurveParameterSpec curve = ECNamedCurveTable.getParameterSpec("secp256k1");
 
+  public static List<String> getPassphrase() {
+    byte[] entropy = new byte[16];
+    List<String> passphrase = new ArrayList<>();
+    try {
+      SecureRandom.getInstanceStrong().nextBytes(entropy);
+      passphrase = MnemonicCode.INSTANCE.toMnemonic(entropy);
+    } catch (NoSuchAlgorithmException | MnemonicLengthException e) {
+      e.printStackTrace();
```
Comment (dafty) [inline src/main/java/io/ark/core/crypto/Crypto.java:38]: Should eventually replace these with SLF4J.

**Review by dafty — APPROVED** [review]

File: src/main/java/io/ark/core/crypto/Crypto.java (hunk 2 above)
Comment (okjintao) [reply src/main/java/io/ark/core/crypto/Crypto.java:38]: yeah, i got a `TODO` to fix the slf4j errors. think it is getting imported in bitcoinj or some other dependency and isn't being resolved propertly

**Comment by boldninja** [comment]
@cdvallejo provide me with your ARK address or contact me on Ark slack (boldninja) if you don't want to share it publicly.

**Comment by okjintao** [comment]
@boldninja APQzKjCzEBkfQNtdarpPMck38CGevaGHBT

**Comment by okjintao** [comment]
@boldninja =)(A82

**Pull request merged**